corehq/apps/app_manager/static/app_manager/js/visit-scheduler.js

```diff
@@ -3,22 +3,38 @@
 var VisitScheduler = (function () {
     'use strict';
 
-    var Scheduler = function (params) {
+    var ModuleScheduler = function(params){
+        // Edits the schedule phases on the module setting page
         var self = this;
-
         self.home = params.home;
-        self.questions = params.questions;
-        self.save_url = params.save_url;
+
+        self.init = function () {
+            _.defer(function () {
+                ko.applyBindings(self, self.home.get(0));
+                self.home.on('textchange', 'input', self.change)
+                // all select2's are represented by an input[type="hidden"]
+                    .on('change', 'select, input[type="hidden"]', self.change)
+                    .on('click', 'a:not(.header)', self.change)
+                    .on('change', 'input[type="checkbox"]', self.change);
+
+                // https://gist.github.com/mkelly12/424774/#comment-92080
```

Comment:
picky, but would be nice to add `//textchange doesn't work with live event binding`